### PR TITLE
Add example jobscript

### LINKED ISSUE #15: PBS scripts erroneously pass data directory as argument to `HealthGPS.Console`
As I changed the config files to use a URL for the data source, if a user also passes in the path to the data folder then they will get an error (I think this may already have bitten @jzhu20). We should update the scripts to drop this argument.

### LINKED ISSUE #17: [ENHANCEMENT] Exemplar HPC script
## Feature request description

Rather than having different pbs scripts floating around everywhere for different projects, we should just have one well-documented generic pbs script.

## Describe the solution you'd like

Should have lots of comments to be a useful reference. Should Contain info about e.g. changing cores, etc.

### LINKED ISSUE #18: `ompthreads` doesn't match `ncpus` in PBS scripts?
https://github.com/imperialCHEPI/healthgps-examples/blob/c2adf449ff63041d08f676d2a7659e0301aff01e/HLM_India/example-india.pbs#L2

@jamesturner246 @jzhu20 I'm working on #17 and just noticed this... Is there any reason why the number of OMP threads is set to much lower than the number of CPUs here? Surely this just means that most CPUs are sitting idle?

### PR DESCRIPTION
**Pull request by alexdewar**
This PR adds an example job script for the Imperial HPC cluster along with instructions on how to use it. I've deleted the old scripts.

The job script is largely the same as the previous ones, except that I've updated the command-line arguments to be correct and it can now be run directly from inside a model config file, without having to download files elsewhere. It no longer copies the input files to ephemeral storage, because I wasn't convinced that is actually necessary. The files are only read in once and even in the worst case they're only tens of MBs in size, which is nothing, really.

@jzhu20 I've tagged you as a reviewer because it'd be good to know if this has everything you need.

Closes #15. Closes #17. Closes #18.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+1. Install [CMake] using the [binary distribution on the CMake website] [^1]
+
+1. Load the build tools
+
+    ```sh
+    module load tools/dev
+    module load GCCcore/12.3.0
+    ```
+
+1. Install [vcpkg] following [these instructions] and set the `VCPKG_ROOT` environment variable to point to the `vcpkg` repo in your `.profile`
```
Comment (jzhu20) [inline README.md:26]: Should we provide an example here?

Comment (alexdewar) [reply README.md:26]: You mean of how to set the `VCPKG_ROOT` variable? It is a bit vague atm...

**Comment by alexdewar** [comment]
Most of the instructions are actually common build steps that people would have to do anyway if they wanted to build HealthGPS. Maybe we should be linking to the HGPS documentation page instead? What do you think @jamesturner246?

File: example-jobscript.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# Model config is in same dir as this script
+config_dir="$(dirname "$0")"
+
+# NB: If you want the config to be configurable with a command-line argument, you can do
+# this instead:
+#   config_dir="$1"
+
+# Load correct modules (be careful when changing).
+module purge
+module load tools/dev
```
Comment (jamesturner246) [inline example-jobscript.sh:25]: It's actually fine to use `tools/prod` here, come to think of it.

```suggestion
module load tools/prod
```

**Review by jamesturner246 — APPROVED** [review]
Agree there's a lot of common steps. Things like vcpkg can just be linked to in the main docs, leaving only specific things like `HGPS_ROOT` here. No need to duplicate info.

**Comment by alexdewar** [comment]
Okey doke. I think what I'll do is I'll make the change suggested by @jzhu20 then merge as is for now so that we can move on to making a release of the examples, but I'll open an issue for linking to the upstream documentation. (On a related note, we might want to give the upstream documentation a bit of love before the release too.)

**Comment by alexdewar** [comment]
Hmm... that merged before the checks passed. The branch protection rules could do with updating.

Would you mind making me an admin so I can tinker with the settings @jamesturner246?

**Comment by jamesturner246** [comment]
Sure, done.

**Pull request merged**